source/user-manual/reference/internal-options.rst

```diff
@@ -1007,6 +1007,12 @@ Vulnerability Detection
 +                                                    +----------------+----------------------------------------------------------------------------------+
 |                                                    | Allowed values | Any integer between 1 and 100000                                                 |
 +----------------------------------------------------+----------------+----------------------------------------------------------------------------------+
+|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for remediations' os data (in number of elements).       |
```

Comment:
```suggestion
|  **vulnerability-detection.remediation_lru_size**  | Description    | LRU cache size assigned for vulnerability remediation (in number of elements).   |
```

Reply:
(Unless `remediations' os data` meant something else.)

Reply:
No, makes sense to change it